qt/widgets/common/inc/MantidQtWidgets/Common/HelpWindow.h

```diff
@@ -8,28 +8,59 @@
 
 #include "MantidQtWidgets/Common/DllOption.h"
 #include <QString>
+#include <QUrl>
+#include <QWidget>
 #include <string>
 
-// forward declarations
-class QUrl;
-
 namespace MantidQt {
 namespace API {
 
-class EXPORT_OPT_MANTIDQT_COMMON HelpWindow {
+/**
+ * HelpWindow provides a centralized interface for displaying
+ * help pages, algorithm details, and concepts. It supports both
+ * the C++ HelpWindow and launching a Python-based HelpWindow.
+ */
+class EXPORT_OPT_MANTIDQT_COMMON HelpWindow : public QWidget {
```

Comment:
What effect does dropping the inheritence from `QWidget` have on available methods/macros?